.github/workflows/node.js.yml

```
@@ -9,9 +9,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Use Node 16
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: 16
```

We should use the opportunity and update node to the highest version supported by GitHub Actions as well:

```suggestion
          node-version: 20
```

---

Sure

---

can you update the node version to node20 in the action.yml as well? that would be great